files/common/var/lib/delphix-platform/ansible/10-delphix-platform/roles/delphix-platform/tasks/main.yml

```diff
@@ -266,23 +278,17 @@
 #
 - set_fact:
     ssh_client_alive_interval: "180"
-  when: platform == "azure"
+    ssh_client_alive_count_max: "0"
+  when:
+    - platform == "azure"
 
 - lineinfile:
     path: /etc/ssh/sshd_config
     regexp: "^#?{{ item.key }} "
     line: "{{ item.key }} {{ item.value }}"
   with_items:
     - { key: "ClientAliveInterval", value: "{{ ssh_client_alive_interval }}" }
-    - { key: "ClientAliveCountMax", value: "0" }
-  when:
-    #
-    # For developer convenience, we only enable the CRA mandated timeout
-    # for external variants. The idle timeout can be a burden when we
-    # need to run long running processes over SSH on our internal
-    # systems (e.g. for development, testing, etc).
-    #
-    - variant is regex("external-.*")
```

Comment:
As this is not linked to CRA, moving this check of external image looks fine.

Reply:
No action expected, resolving.